### PR TITLE
Fixes redirect to Data Converter

### PR DESCRIPTION
**Pull request by aarohib**
## What does this PR do?
Updates the security guide with a data converter page with links to all the relevant topics.

### CODE REVIEW

File: docs-src/security/data-converter.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+id: data-converter
+title: Data Converter
+sidebar_label: Data Converter
+description: Use a Data Converter with a custom Payload Codec to encode and decode your data.
+tags:
+  - guide-context
+---
+
+Temporal SDKs provide a [Data Converter](/concepts/what-is-a-data-converter) that can be customized with a custom [Payload Codec](/concepts/what-is-a-payload-codec) to encode and secure your data.
```
Comment (djmagee) [inline docs-src/security/data-converter.md:10]:
```suggestion
Each Temporal SDK provides a [Data Converter](/concepts/what-is-a-data-converter) that can be customized with a custom [Payload Codec](/concepts/what-is-a-payload-codec) to encode and secure your data.
```

**Review by djmagee — APPROVED** [review]
One language tweak, and it's good to go.

File: docs-src/security/data-converter.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+title: Data Converter
+sidebar_label: Data Converter
+description: Use a Data Converter with a custom Payload Codec to encode and decode your data.
+tags:
+  - guide-context
+---
+
+Each Temporal SDK provides a [Data Converter](/concepts/what-is-a-data-converter) that can be customized with a custom [Payload Codec](/concepts/what-is-a-payload-codec) to encode and secure your data.
+
+For details on what data can be encoded, how to secure it, and what to consider when using encryption, see [Data encryption](/prod-readiness-context/data-encryption).
```
Comment (lorensr) [inline docs-src/security/data-converter.md:12]: @aarohib this link doesn't work. Directs to `https://production-readiness/develop#data-encryption`

Comment (aarohib) [reply docs-src/security/data-converter.md:12]: dang; fixing.

**Pull request merged**